pysrc/SBProfile.cpp

```diff
@@ -79,6 +79,9 @@ namespace galsim {
                 .def(bp::self == bp::other<GSParams>())
                 .enable_pickling()
                 ;
+#if BOOST_VERSION >= 106000 && BOOST_VERSION < 106100
```

Comment:
OK.  Thanks for checking that.  Maybe just add a comment in the code then to say that this is a workaround for what we believe is a bug in a single version of boost that was fixed in 1.61.  And reference this thread for more details.  Just in case it crops up again.